slasher/cache/cache.go

```diff
@@ -0,0 +1,73 @@
+package cache
```

Comment:
perhaps make the name `cache.go` more specific, when person B comes in and implements a new cache for other functionality, they don't have to worry about changing both